config/kube.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+
+import click
+import os
+import sys
+import syslog
+import urllib3
+import tempfile
+import requests
+import fcntl
+import sonic_device_util
+import yaml
+import netaddr
+import inspect
+import shutil
+from urlparse import urlparse
+from swsssdk import ConfigDBConnector
+
+def hostname():
+    return os.uname()[1]
+
+from config.main import run_command, AbbreviationGroup, config
+
+logger_identity = "kube_config"
+log_level = syslog.LOG_DEBUG
+
+KUBE_ADMIN_CONF = "/etc/sonic/kube_admin.conf"
+KUBELET_YAML = "/var/lib/kubelet/config.yaml"
+KUBELET_SERVICE = "/etc/systemd/system/multi-user.target.wants/kubelet.service"
+
+SERVER_ADMIN_URL = "https://{}/admin.conf"
+KUBEADM_JOIN_CMD = "kubeadm join --discovery-file /etc/sonic/kube_admin.conf --node-name {}"
+
+LOCK_FILE = "/var/lock/kube_join.lock"
+
+def _log_msg(lvl, fname, ln, m):
+    if lvl <= log_level:
+        syslog.openlog(logger_identity)
+        syslog.syslog(lvl, m)
+        syslog.closelog()
+
+    if (
```

Comment:
>( [](start = 7, length = 1)

Make first condition in the same line of `(` or make `)` below last condition

Reply:
done